README.rst

```diff
@@ -8,12 +8,22 @@ This project contains branding assets and themes for edx.org. It is the edX impl
 Usage
 -----
 
-Install this package:
+Install this package one of two ways
```

Comment:
Can we add any guidance here on why you might prefer one or the other?

Reply:
Added!